patches/chrome-installer-linux-rpm-build.sh.patch

```diff
@@ -13,7 +13,25 @@ index 0bcd8689d45850ad539f99423fa211785db4f343..9a2d8eeb97e096db8b80aac7e2600688
      local INSTALLDIR="${INSTALLDIR}-${CHANNEL}"
      local PACKAGE="${PACKAGE}-${CHANNEL}"
      local MENUNAME="${MENUNAME} (${CHANNEL})"
-@@ -108,7 +109,10 @@ do_package() {
+@@ -27,7 +28,6 @@ gen_spec() {
+ # Setup the installation directory hierachy in the package staging area.
+ prep_staging_rpm() {
+   prep_staging_common
+-  install -m 755 -d "${STAGEDIR}/etc/cron.daily"
+ }
+ 
+ # Put the package contents in the staging area.
+@@ -49,9 +49,6 @@ stage_install_rpm() {
+   SHLIB_PERMS=755
+   stage_install_common
+   log_cmd echo "Staging RPM install files in '${STAGEDIR}'..."
+-  process_template "${BUILDDIR}/installer/common/rpmrepo.cron" \
+-    "${STAGEDIR}/etc/cron.daily/${PACKAGE}"
+-  chmod 755 "${STAGEDIR}/etc/cron.daily/${PACKAGE}"
```

Comment:
This patch builds the rpm, but it might not work as desired, I'll test that shortly:

```
diff --git a/patches/chrome-installer-linux-rpm-build.sh.patch b/patches/chrome-installer-linux-rpm-build.sh.patch
index 0ed44a13..163d51d1 100644
--- a/patches/chrome-installer-linux-rpm-build.sh.patch
+++ b/patches/chrome-installer-linux-rpm-build.sh.patch
@@ -1,5 +1,5 @@
 diff --git a/chrome/installer/linux/rpm/build.sh b/chrome/installer/linux/rpm/build.sh
-index 0bcd8689d45850ad539f99423fa211785db4f343..9a2d8eeb97e096db8b80aac7e26006888200e9f8 100755
+index 0bcd8689d45850ad539f99423fa211785db4f343..36e080915c22436da9f3ed8137cc6ac8338b5166 100755
 --- a/chrome/installer/linux/rpm/build.sh
 +++ b/chrome/installer/linux/rpm/build.sh
 @@ -15,8 +15,9 @@ gen_spec() {
@@ -13,7 +13,15 @@ index 0bcd8689d45850ad539f99423fa211785db4f343..9a2d8eeb97e096db8b80aac7e2600688
      local INSTALLDIR="${INSTALLDIR}-${CHANNEL}"
      local PACKAGE="${PACKAGE}-${CHANNEL}"
      local MENUNAME="${MENUNAME} (${CHANNEL})"
-@@ -108,7 +109,10 @@ do_package() {
+@@ -52,6 +53,7 @@ stage_install_rpm() {
+   process_template "${BUILDDIR}/installer/common/rpmrepo.cron" \
+     "${STAGEDIR}/etc/cron.daily/${PACKAGE}"
+   chmod 755 "${STAGEDIR}/etc/cron.daily/${PACKAGE}"
++  rm -rf "${STAGEDIR}/etc/cron.daily"
+ }
+ 
+ verify_package() {
+@@ -108,7 +110,10 @@ do_package() {
      --define "${COMPRESSION_OPT}" \
      --define "__os_install_post  %{nil}" \
      "${SPEC}"
@@ -25,7 +33,7 @@ index 0bcd8689d45850ad539f99423fa211785db4f343..9a2d8eeb97e096db8b80aac7e2600688
    mv "$RPMBUILD_DIR/RPMS/$ARCHITECTURE/${PKGNAME}.${ARCHITECTURE}.rpm" \
       "${OUTPUTDIR}"
    # Make sure the package is world-readable, otherwise it causes problems when
-@@ -145,7 +149,10 @@ verify_channel() {
+@@ -145,7 +150,10 @@ verify_channel() {
        CHANNEL=stable
        ;;
      unstable|dev|alpha )
```

Reply:
Looking closer, I can see that the problem is that [spec file expects `/etc/` to be present](https://cs.chromium.org/chromium/src/chrome/installer/linux/rpm/chrome.spec.template?sq&g=0&l=65) and so removing it without removing this copy will break the build, as you discovered.

Since there is nothing other than the undesirable cronjob that needs to be put in `/etc/ at build time, we can get rid of that directory and that copy entirely.